src/selectedtests/task_mappings/mappings.py

```diff
@@ -142,9 +140,9 @@ def _get_filtered_files(diff: DiffIndex, regex: Pattern) -> List[str]:
     :return: A list of the changed files that matched the given regex pattern.
     """
     re: List[str] = []
-    for file in _get_changed_files(diff):
-        if match(regex, file.b_path):
-            re.append(file.b_path)
+    for file in get_changed_files(diff, LOGGER):
```

Comment:
@dbradf I'm not super familiar with logging in python or structlog but it looks like here https://github.com/dbradf/evgflip/blob/master/src/evgflip/find_flips.py that you're getting the logger instance that you initialized when the cli first ran. Is that correct?

If we can do so, it'd be great if we could remove the ```LOGGER``` param here and just get the logger instance directly in ```git_helper.py```. It would cut down on the params needed by the function and make ```get_changed_files``` a bit more reusable.

Reply:
The big benefit of passing a logger this way as opposed to getting a logger instance in `git_helper.py` is that structlog has the concept of [binding loggers](http://www.structlog.org/en/stable/getting-started.html#building-a-context), which allows you to assign variables to the logger that are printed out with all log entries. 

We aren't using it right now, but it could potentially be useful. For example, in this project it might be useful to bind the project_id to the logger up front and then every log entry made with that logger will include the project_id. When looking through logs that have a lot going on (especially if there are multiple threads or multiple processes), that can be extremely valuable.

So, either method works, it just depends on what we want. Since this project is just getting started, it doesn't really matter. It should be easy to change later.

Reply:
Sounds good to me